courseraresearchexports/commands/jobs.py

```diff
@@ -0,0 +1,229 @@
+import argparse
+
+
+def download(args):
+    """
+    Download research export job using a job id.
+    """
+    job = api.get(args.id)
+    utils.download_export(job, args.dest)
+
+
+def create(args):
+    """
+    Creates a post request to Coursera's research exports service
+     with specified parameters
+    """
+    export_job_json = utils.create_export_job_json(**vars(args))
+    export_job_id = api.create(export_job_json)
+
+    print('Successfully created job {}:\n{}'.format(
+        export_job_id,
+        json.dumps(export_job_json, indent=2)))
+
+
+def get(args):
+    """
+    Get status of a research export job using a job id.
+    """
+    job = api.get(args.id)
+
+    scope_id, scope_name = utils.get_scope_id_and_name_from_job(job)
+
+    export_type = job['exportType']
+
+    if export_type == 'RESEARCH_WITH_SCHEMAS':
+        schemas_str = 'Schemas: ' + ', '.join(job['schemaNames']) + '\n'
+    else:
+        schemas_str = ''
+
+    creation_time = datetime.fromtimestamp(
+            job['metadata']['createdAt']/1000.0).strftime('%c')
+
+    download_link = job['downloadLink'] if 'downloadLink' in job else None
+
+    job_str = 'Job id: {}\nScope id: {}\nScope name: {}\nType: {}\n' \
+              'Created: {}\nStatus: {}\n{}Download Link: {}'
+
+    print(job_str.format(
+        job['id'],
+        scope_id,
+        scope_name,
+        job['exportType'],
+        creation_time,
+        job['status'],
+        schemas_str,
+        download_link))
+
+
+def get_all(args):
+    """
+    Get status of all research export jobs that you have requested.
+    """
+    export_jobs = api.get_all()
+
+    # print out jobs by creation date
+    template = "{0:12}{1:12}{2:10}{3:25}{4:12}{5:18}"
+    print(template.format("Time", "ExportID", "Type",
+                          "Scope", "Status", "Schemas"))
+    for job in sorted(
+            export_jobs,
+            key=lambda x: x['metadata']['createdAt'],
+            reverse=True):
+
+        creation_time = datetime.fromtimestamp(
+            job['metadata']['createdAt']/1000.0).strftime('%Y-%m-%d')
+
+        if job['exportType'] == 'RESEARCH_EVENTING':
+            schema_names = 'events'
+        elif job['exportType'] == 'GRADEBOOK':
+            schema_names = 'gradebook'
+        elif len(job['schemaNames']) == len(SCHEMA_NAMES):
+            schema_names = 'all'
+        else:
+            schema_names = ','.join(job['schemaNames'])
+
+        if job['exportType'] == 'RESEARCH_WITH_SCHEMAS':
+            export_type = 'schemas'
+        elif job['exportType'] == 'RESEARCH_WITH_EVENTING':
+            export_type = 'eventing'
+        else:
+            export_type = 'gradebook'
+
+        _, scope_name = utils.get_scope_id_and_name_from_job(job)
+
+        print(template.format(
+            creation_time,
+            job['id'],
+            export_type,
+            scope_name,
+            job['status'],
+            schema_names))
+
+
+def parser(subparsers):
```

Comment:
do you think it makes sense to split the subparser up depending on the export type? (e.g 1 subparser for table export, and another for clickstream) 

that way all the flags specific to 1 type of export is guaranteed to make sense.